src/App.jsx

```diff
@@ -14,45 +15,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
+
+const sortGoods = (array, sortParameter, reverseParameter) => {
+  switch (sortParameter) {
+    case SORT_FIELD_ALPHABETICALLY:
+      // eslint-disable-next-line no-param-reassign
+      array = [...array].sort((a, b) => a.localeCompare(b));
+      break;
```

Comment:
Don't disable eslint. Try to fix errors instead. 
```suggestion
const sortGoods = (goods, sortParameter, reverseParameter) => {
 const preparedGoods = [...goods];
 
  switch (sortParameter) {
    case SORT_FIELD_ALPHABETICALLY:
      preparedGoods.sort((a, b) => a.localeCompare(b));
      break;
```